### DISCUSSION TITLE
Question about supported utilities

### DISCUSSION THREAD

**Discussion by artem-tarasenko** [jaredh159/tailwind-react-native-classnames#136] [Q&A] [answered]
Hi, I'm wondering is the file "supported utilities.md" could be referenced about what could be used or not?
If yes , would there be any work to extend the list? Would love to see more stuff available in the future, thanks for the effort.

**Comment by jaredh159** [comment, marked as answer]
Thanks for the question.

I don't reference the `supported-utilities.md` file more prominently because the support for RN (_Native_) is pretty complete. The only hole I'm aware of at the moment is [transform support](https://github.com/jaredh159/tailwind-react-native-classnames/issues/106). 

Now, if you're using RN _Web_, there's a ton of missing utilities. But, the goal (right now at least) of this library is only to support RN-native style props.  Are you targeting RN web?  We have a [discussion going on that](https://github.com/jaredh159/tailwind-react-native-classnames/discussions/125) if you're interested. Or am I misunderstanding the question?

**Reply by artem-tarasenko** [reply]
Yeah, I gussed right, I'm using expo so web is one of the targeted platforms. Looks like I did a wrong assumption about the lib without checking :) Will rethink my usage of it, thanks for replying.
